pkg/apis/packet/v1alpha1/types_controlplane.go

```diff
@@ -24,4 +24,11 @@ import (
 // ControlPlaneConfig contains configuration settings for the control plane.
 type ControlPlaneConfig struct {
 	metav1.TypeMeta `json:",inline"`
+
+	Persistence *Persistence `json:"persistence,omitempty"`
```

Comment:
doc string missing

Also, it seems to be an optional field, so the `// +optional` tag should be added

Reply:
Please update https://github.com/gardener/gardener-extension-provider-packet/blob/master/docs/usage-as-end-user.md#controlplaneconfig and https://github.com/gardener/gardener-extension-provider-packet/blob/master/example/30-controlplane.yaml

Reply:
Got the first comment. Not sure about the second, please look.